src/components/people/PersonPositionList/PersonPositionCards.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { PersonPosition } from '@equinor/fusion';
+import { PersonPositionCard } from '@equinor/fusion-components';
+import * as styles from './styles.less';
+
+type PersonPositionCardsProps = {
+    positions: PersonPosition[];
+    disableOrgLink?: boolean;
+};
+
+type PositionProject = {
+    projectName: string;
+    id: string;
+    positions: PersonPosition[];
+};
+
+type CardLinkProps = {
+    position: PersonPosition;
+    disableOrgLink: boolean;
+};
+
+const CardLink: React.FC<CardLinkProps> = ({ children, disableOrgLink, position }) => {
```

Comment:
think you need to wrap the `CardLinkProps` with `PropsWithChildren<CardLinkProps>`
it works, but guessing the linter will complain.
then simply return children without the fragment wrapper

Reply:
After adding PropsWithChildren the linter still complains if I remove the fragment